tests/workflows/test_basin_mask.py

```diff
@@ -1,123 +1,18 @@
 # -*- coding: utf-8 -*-
 """Tests for the hydromt.workflows.basin_mask."""
 import logging
-import os
 
-import geopandas as gpd
 import numpy as np
 import pytest
-import xarray as xr
 
 import hydromt
-from hydromt import DataCatalog, raster
-from hydromt.models import MODELS
+from hydromt import raster
 from hydromt.workflows.basin_mask import (
     _check_size,
-    _parse_region_value,
     get_basin_geometry,
-    parse_region,
 )
 
-logger = logging.getLogger("tets_basin")
-
-
-@pytest.mark.skip(reason="Needs refactor to Pydantic BaseModel.")
-def test_region(tmpdir, world, geodf, rioda):
```

Comment:
But you did move them all to test_model_region.py even if they are now skipped right?

Reply:
The relevant ones yes. I think the one that test `parse_region_value` I omitted because these are now done by pydantic directly and therefore don't have to be tested there, but the coverage should be equivalent.